_pages/rules.html

```diff
@@ -64,7 +64,7 @@ <h2 class="page-heading">C. Client Mods</h2>
           </ul>
           <li>For any other mods, email <a href="mailto:brott@oc.tc">brott@oc.tc</a> for confirmation before using it.</li>
           <span class="badge badge-warning">Note</span>
-          <small>Unless explicitly permitted, additional mods which can be installed through LabyMod are not allowed.</small>
+          <small>Unless explicitly permitted, additional mods which can be installed through LabyMod that affect gameplay are not allowed. When installing LabyMod, you must not install its Damage Indicator mod.</small>
```

Comment:
I find this redundant. Does Labymod have damage indicators by default?

Reply:
![image](https://user-images.githubusercontent.com/43831917/108421510-214e5600-71ea-11eb-99fe-61d9d8ec301e.png)

LabyMod has a few mods you can install out of the box, with several others available as downloadable mods in game. It's one of the mods that is available to add during install, but perhaps we do not need to add a note if we have the note above (I had misremembered it being selected by default).

Reply:
I think that is what is referred to with "additional mods which can be installed through LabyMod are not allowed".

Reply:
My reasoning behind the further clarifications lies in the fact that the vast majority of LabyMod's optional mods are features that exist in Badlion Client and Lunar Client, which are clients that are explicitly permitted. For example, mods like ItemPhysics, Laby's Keystrokes, Resourcepacks24, More Particles etc. that are available to add when installing LabyMod itself are mostly cosmetic or for adding extra info on the HUD during play that exist in both/either Badlion and Lunar, while others that may not be permitted are only enabled on servers if the server explicitly seeks out to permit it, such as Laby's Minimap. Other mods which you can download within the client, such as Custom Time or Full Bright, or Better Perspective are all features integrated into Badlion and Lunar. 

This being said I don't think I'm happy anymore with how I've worded it in this PR. I do think how it's worded previously might be confusing for some, given that Badlion and Lunar are explicitly permitted with many of the same features integrated as opposed to available for download after installation.